content/backmatter.tex

```diff
@@ -334,6 +334,7 @@ \section{Overview}\label{subsec:dep_overview}
     & \shortstack{\textbf{Last Version Supported}}
     & \textbf{Replaced By} \\
     \hline
+    Header Directory: \hyperref[subsec:dep_rationale:mpp]{\HEADER{mpp}} & 1.1 & Current & (none) \\ \hline
```

Comment:
Actually, I don't know about the extensions. It might be better to not include them in the mpp directory so as to force implementations that are using `<mpp/shmemx.h>` to side-grade to `<shmemx.h>`. The intent was backwards compatibility with SGI SHMEM, which did not have a `shmemx.h` header.